### PR TITLE
mc

### PR DESCRIPTION
**Pull request by ValeraFinebits**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16142)

### CODE REVIEW

File: manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml [hunk 1]
```diff
@@ -10,16 +10,17 @@ Tags:
...
+- files
+- folders
+- explorer
 ShortDescription: Metro Commander modern two panes file manager.
 PackageUrl: https://www.microsoft.com/store/apps/9WZDNCRFJB33
 Installers:
 - Architecture: neutral
   InstallerUrl: https://finebitsstorage.z22.web.core.windows.net/metrocommander/packages/MC_1.3.86.0.msix
   InstallerSha256: C30BC2FC1595D4A284C4350A63E7FD74DDDDA44C64D739DB54297B2C4E9D364A
   InstallerType: msix
```
Comment (ItzLevvie2) [inline manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml:23]:
```suggestion
  InstallerType: msix
  InstallerSwitches:
    Silent: /S
    SilentWithProgress: /S
```

Comment (ItzLevvie) [inline manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml:23]:
```suggestion
  InstallerType: msix
  InstallerSwitches:
    Silent: /S
    SilentWithProgress: /S
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my changes; all manifests must install the application in unattended mode.

**Comment by ValeraFinebits** [comment]
Please tell me what the problem is. There's nothing in the logs.

File: manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/f/Finebits/MetroCommander/1.3.86.0/Finebits.MetroCommander.yaml]: Looks like I overlooked this (my brain thought that `msix` was `msi`), but the switches actually aren't needed here.

**Comment by ChuckFerring** [comment]
Installation fails because of a dependency on the package Microsoft.Advertising.Xaml. Here is the tail of the winget installation log:

2021- 6-18 12:40:38.949 [CORE] Deployment failed #0: Windows cannot install package BOOSTUDIOLLC.METROCOMMANDER_1.3.86.0_x64__b6e429xa66pga because this package depends on a framework that could not be found. Provide the framework "Microsoft.Advertising.Xaml" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 10.1811.1.0, along with this package to install. The frameworks with name "Microsoft.Advertising.Xaml" currently installed are: {Microsoft.Advertising.Xaml_10.1808.3.0_x64__8wekyb3d8bbwe}
2021- 6-18 12:40:38.949 [FAIL] C:\BA\268\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(50)\AppInstallerCLI.exe!00007FF6FF19E1BD: (caller: 00007FF6FF19E898) Exception(1) tid(ce8) 80073CF3 Package failed updates, dependency or conflict validation.

    Msg:[Install failed: Windows cannot install package BOOSTUDIOLLC.METROCOMMANDER_1.3.86.0_x64__b6e429xa66pga because this package depends on a framework that could not be found. Provide the framework "Microsoft.Advertising.Xaml" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 10.1811.1.0, along with this package to install. The frameworks with name "Microsoft.Advertising.Xaml" currently installed are: {Microsoft.Advertising.Xaml_10.1808.3.0_x64__8wekyb3d8bbwe}] 

2021- 6-18 12:40:38.952 [CLI ] MSIX installer failed: 2147958003
2021- 6-18 12:40:38.952 [CLI ] Terminating context: 0x80073cf3 at C:\BA\268\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:15d

**Pull request closed without merging**